lib/std/posix.zig

```diff
@@ -173,13 +173,13 @@ pub const W_OK = system.W_OK;
 pub const X_OK = system.X_OK;
 
 pub const iovec = extern struct {
-    iov_base: [*]u8,
-    iov_len: usize,
+    ptr: [*]u8,
```

Comment:
The naming convention is to keep the posix name, but remove the prefix. I understand that this type fulfills the same purpose as a Zig language slice, however, I don't see why that means an exception to the naming convention.

Reply:
I decided to use the zig naming here made more sense because the primary goal was to be able to interop between posix's iovec and windows's WSABUF. posix calls it `base`, windows calls it `buf`, so I decided to adopt the zig name here.

Would you prefer to use `base` for posix.iovec and WSABUF instead of `ptr`?

Reply:
WSABUF should continue to use `len` and `buf`, matching the API definitions in windows.h, and iovec should continue to use `base` and `len`, matching the POSIX API definitions.

The cross-platform abstractions offered by the zig standard library must be implemented on top of that.